5calls/app/src/main/res/layout/activity_stats.xml

```diff
@@ -46,6 +46,20 @@
                 style="@style/graphViewStyle"
```

Comment:
Woops, looks like GraphView wasn't properly removed in the rebase, so now there's a crash. I'll fix it!

Reply:
I am sorry to heard that. Is there something I can help with?

Reply:
I fixed it. Next time you might want to try building before uploading it, and I'll also remember to build it before accepting it.